src/core/Texture.tsx

```diff
@@ -1,7 +1,6 @@
-import * as React from 'react'
```

Comment:
I think we need this since we're not using automatic JSX pragma. 

I would re-add this back & we can look at solving the real issue at hand later.

Reply:
Yep exactly, we can probably solve this quite simply, but imo OOS for this PR...

Reply:
@joshuaellis do i need to add it back?

Reply:
@clementroche yes :)

Reply:
done